### DISCUSSION TITLE
Deleting Scenes with Personal media content

### DISCUSSION THREAD

**Discussion by H-Gungor** [mozilla/hubs#4111] [General Questions & Discussion]
Hi,

I created a scene on Spoke and published on Hubs, however I wasn't aware that I can't delete the scenes (or can I?) and included personal media files that shouldn't be shared publicly yet and also need to make further changes on the scene. It was more like a trial. How can I have the published scene deleted or unpublished (without deleting the Spoke project)? I also read that deleting the Project doesn't delete the published scenes. Is that also correct?

Many thanks!

**Comment by no-disassemble** [comment]
Just wanted to share my thoughts surrounding this feature. There is no way to prototype a scene unless it is published. Deleting scenes from "My Scenes" is a critical feature that needs to be added. At the very least, users should be informed when publishing a scene that it can never be deleted.

**Reply by H-Gungor** [reply]
Thanks for your reply. I definitely agree, if I had known, I would have added certain content only in the hubs, not in Spoke, and I even read the instructions and some articles before I started, still didn't notice I can't unpublish or delete; at least an unpublish or edit option would be lifesaver! It must be causing so much clutter for them too as I doubt people would get it right with the first one so I thought I must be missing a Delete button somewhere.. 

I hope they look into it as it can be really problematic in terms of privacy..
